protobuf/stream.pb.go

```diff
@@ -51,6 +52,13 @@ func (m *Request) GetEvents() []*EventType {
 	return nil
 }
 
+func (m *Request) GetForceClose() bool {
```

Comment:
__[golint]__
* [golint] exported method Request.GetForceClose should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->